xof.go

```diff
@@ -0,0 +1,44 @@
+package kyber
+
+import (
+	"crypto/cipher"
+	"io"
+)
+
+// An XOF is an extendable output function, which is a cryptographic
+// primitive that can take arbitrary input like a hash, and then create
```

Comment:
I'd remove `like a hash` - it made me think that the input should be a hash.

Reply:
Fixed to be clearer.